api/proto/info/info.proto

```diff
@@ -405,6 +405,31 @@ enum ZDeviceState {
     ZDEVICE_STATE_BOOTING = 5;
 }
 
+enum StorageRaidType {
+  STORAGE_RAID_TYPE_UNSPECIFIED = 0;
+  STORAGE_RAID_TYPE_RAID_ZERO = 1;         //RAID-0
+  STORAGE_RAID_TYPE_RAID_MIRROR = 2;       // RAID-1
+  STORAGE_RAID_TYPE_RAIDZ = 3;             // raidz (RAID-5)
+  STORAGE_RAID_TYPE_RAIDZ_TWO = 4;         // raidz2 (RAID-6)
+  STORAGE_RAID_TYPE_RAIDZ_THREE = 5;       // raidz3 (RAID-7)
+  STORAGE_RAID_TYPE_NORAID = 6;            // without RAID
+}
+
+enum StorageTypeInfo {
+  STORAGE_TYPE_INFO_UNSPECIFIED = 0;
+  STORAGE_TYPE_INFO_EXT_FS = 1;       // ext4
```

Comment:
Would be more consitent to call this STORAGE_TYPE_INFO_EXT4

Reply:
corrected